fs/fs.go

```diff
@@ -137,16 +138,28 @@ func FolderExists(folderPath, name string) bool {
 
 // CopyFile copy a file or folder from one path to another
 func CopyFile(origFilePath, destFilePath string) error {
-	input, err := os.ReadFile(origFilePath)
-	if err != nil {
+	var src, dest *os.File
+	var err error
+
+	if src, err = os.Open(origFilePath); err != nil {
+		return err
+	}
+	defer src.Close()
+
+	if dest, err = os.Create(destFilePath); err != nil {
 		return err
 	}
+	defer dest.Close()
 
-	err = os.WriteFile(destFilePath, input, rwFilePermission)
+	_, err = io.Copy(dest, src)
 	if err != nil {
 		return err
 	}
 
```

Comment:
Actually can you add `err = dest.Sync()` to be sure the copied file is not corrupted ?

Reply:
Now we force to flush data, so it is not necessary anymore!